azurerm/import_arm_operational_insight_workspace_test.go

```diff
@@ -0,0 +1,56 @@
+package azurerm
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccAzureRMOperationalInsightWorkspace_importrequiredOnly(t *testing.T) {
+	resourceName := "azurerm_operational_insight_workspace.test"
+
+	ri := acctest.RandInt()
+	config := testAccAzureRMOperationalInsightWorkspace_requiredOnly(ri, testLocation())
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMOperationalInsightWorkspaceDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+			},
+
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func TestAccAzureRMOperationalInsightWorkspace_importoptional(t *testing.T) {
```

Comment:
_minor_ we'd tend to name this `_importOptional` - could we update it to match?

Reply:
Done